### DISCUSSION TITLE
literally no play store, cant download

### DISCUSSION THREAD

**Discussion by cynthixcbc** [Genymobile/feedback#41] [Genymotion Desktop] [answered]
s

**Comment by pgivel** [comment, marked as answer]
Hi @cynthixcbc :) 
Have you tried using the [Open GApps widget](https://docs.genymotion.com/desktop/04_Emulating_sensors_and_features/#open-gapps)? It installs on your virtual device a minimal package of the Google Play Services, including the Google Play Store.

**Comment by cynthixcbc** [comment]
Thank you so much!
